features/step_definitions/admin_steps.rb

```diff
@@ -340,3 +340,11 @@ def update_settings(settings)
   u = User.find_by_login(user)
   u.update_attribute(:banned, false)
 end
+
+When(/^the last login information has been updated$/) do
+    User.update_last_login
+end
+
+Then(/^I should see the last login time for "(.*?)"$/) do |arg1|
```

Comment:
Unused block argument - `arg1`. You can omit the argument if you don't care about it.